kubejs/server_scripts/mod_specific/ATM/atm_shard.js

```diff
@@ -0,0 +1,26 @@
+onEvent('recipes', e => {
+  let shard = ('allthetweaks:atm_star_shard')
+
+//#Pipez
+  e.shaped('16x pipez:infinity_upgrade', ['ABA', 'BCB', 'ADA'], {

```

Comment:
Infinity upgrades should not be craftable

Reply:
ah, we have that in 7 as well, want it removed from there too?, and is that just for Pipez?

Reply:
Yes recipe should be removed from 7 as well - infinity upgrade is just too laggy